### PR TITLE
JS: Enhancing digitizing component and module

### PR DESCRIPTION
**Pull request by rldhont**
* Add erase all button and method
* Removed from localstorage when saved is toggled
* Add a measure attribute to enable or not the measure button
* Fix locale key confirm erase by replacing confirme by confirm

### CODE REVIEW

**Review by Gustry — COMMENTED** [review]
If all buttons have a nice UI (tooltip, CSS, position, icon), LGTM

**Comment by rldhont** [comment]
@Gustry @nboisteault it is ready for review

File: lizmap/app/locales/cs_CZ/view/locales/dictionnary.UTF-8.properties [hunk 1]
```diff
@@ -192,7 +192,7 @@ digitizing.toolbar.textScale=Velikost
 digitizing.toolbar.import=Importovat v GeoJSON, GPX nebo KML
 digitizing.toolbar.edit=Upravte ( podržte klávesu "Shift" pro výběr více funkcí)
 digitizing.toolbar.erase=Smazat některé funkce
-digitizing.confirme.erase=Opravdu chcete smazat tuto funkci?
```
Comment (nboisteault) [inline lizmap/app/locales/cs_CZ/view/locales/dictionnary.UTF-8.properties:195]: I think this fix must only be made in lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties

@Gustry can you confirm?

Comment (Gustry) [reply lizmap/app/locales/cs_CZ/view/locales/dictionnary.UTF-8.properties:195]: I think he is right, therefore translations are not lost.
It's not translators fault if we rename the key.

**Pull request merged**